client-src/elements/chromedash-feature-page.js

```diff
@@ -452,14 +468,16 @@ export class ChromedashFeaturePage extends LitElement {
     return html`
       ${this.loading ?
         this.renderSkeletons() :
-        html`
-          ${this.renderSubHeader()}
-          <div id="feature">
-            ${this.renderFeatureContent()}
-            ${this.renderFeatureStatus()}
-          </div>
-          ${this.renderFeatureDetails()}
-      `}
+        this.isFeatureLoaded() ?
```

Comment:
These conditionals would probably be more clear as regular if-statements with return html statements for each case.  E.g.:
if (this.loading) {
  return this.renderSkeletons();
}
if (this.isFeatureLoaded()) {
  return html...
 ;
}
return '';